### ISSUE TITLE
if `let g:gutentags_enabled = 0` then :GutentagsUpdate throws error

### ISSUE THREAD

**Issue by Konfekt** [ludovicchabant/vim-gutentags#19]
If automatic tag generation is disabled then either `:GutentagsUpdate ` should not be defined  or still work (to update tags manually).

**Comment by ludovicchabant** [comment]
Can't repro... maybe pull latest and try again?
What's the error being thrown?

**Comment by Konfekt** [comment]
Pulled latest, but it still throws 
```vim
E119: Not enough arguments for function: <SNR>255_manual_update_tags
```

**Comment by ludovicchabant** [comment]
That's weird. I fixed the signature of this method in 981bcd8d45133d987cc3be77d7d094e9ffe5bd1c but AFAIK there's only [one call site](https://github.com/ludovicchabant/vim-gutentags/blob/master/autoload/gutentags.vim#L127) for it and it only passes one argument. Can you check your code looks the same?

**Comment by Konfekt** [comment]
Hm.. pulling latest fixed it. Thanks!
